src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -167,11 +173,72 @@ export class DatePicker {
   handleDateChange(ev) {
     ev.stopPropagation();
     this._input.value = ev.detail;
-    this.value = ev.detail;
+    if (this.time) {
+      const chiTime = this.el.querySelector('.chi-popover__content chi-time') as HTMLElement;
+      const valueTime = chiTime.getAttribute('value');
+
+      if (valueTime) {
+        const time = valueTime.split(':');
+        const period = parseInt(time[0]) >= 12 ? 'pm' : 'am';
+        const hours = parseInt(time[0]) >= 12 ? parseInt(time[0]) - 12 : parseInt(time[0]);
```

Comment:
I've seen other checks in which you compare time > 12 instead of time >= 12..Which is the correct one?